roles/puma/templates/puma.service

```
@@ -1,14 +1,16 @@
 [Unit]
 Description=Puma HTTP Server for {{ app_name }} ({{ env }})
 After=network.target
+Requires=puma_{{ app_name }}_{{ env }}.socket
```

Would this work as well?

```suggestion
Requires={{ puma_service_unit_name }}.socket
```